pkg/tnf/reel/reel.go

```diff
@@ -210,10 +210,12 @@ func (r *Reel) Step(step *Step, handler Handler) error {
 
 				output, outputStatus := r.stripEmulatedPromptFromOutput(result.Output)
 				if outputStatus != 0 {
-					r.Err = fmt.Errorf("error executing command %d: ", outputStatus)
+					r.Err = fmt.Errorf("error executing command exit code:%d", outputStatus)
+					return r.Err
```

Comment:
I think the right thing to do is continue here and return r.Err outside the loop. There seems to be other cases where r.Err get set. It's wrong to ignore it for the last step, which in our case, the only step most of the time.

Reply:
fixed